src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java

```diff
@@ -507,6 +522,10 @@ public String getRbacPassword() {
         return rbacPassword;
     }
 
-    public String getApiKey() { return apiKey; }
     public String getEnvironment() { return environment; }
+
+    public String getApiKey(final String countryCode) {
+        final String apiKey = countryToApiKeyMap.get(countryCode);
+        return StringUtils.isEmpty(apiKey) ? "KEY_NOT_FOUND" : apiKey;
```

Comment:
Are we using this `KEY_NOT_FOUND` somewhere ?
I am assuming it yes. So why not create a constant for it ?

Reply:
Updated in new PR#6